### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Container Table_Tables_Screenreader: Screen reader is announcing the entire row information at once when focus lands on the first table row.

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6346]
**GitHub Tags:**
#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2022;#Azure Tools Storage Explorer;#FTP;#Screenreader;#WCAG1.3.1;#NVDA;#Element:Table;

## Actual Result: 
Screen reader is announcing the entire row information at once when focus lands on the first table row. it's announcing the header information more than twice.

**Issue is not observed with jaws**
## Expected Result:
Screenreader should not announce the entire row information at once when focus lands on the table row. Screen reader should announce the data items descriptively when user navigates using ctrl+alt+arrow key and it should not announce the header information more than twice.
## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
NVDA version 2020.4
JAWS version 2021.205.53
## OS Details:
Microsoft Windows 11 Enterprise
version 22H2 OS Build 22621.819

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Turn on NVDA
7. Navigate to Table present in right side of the storage explorer.
8. Tab till the row and verify whether the screen reader is announcing the entire table row information at once or not.
### User Impact:
Screen reader users may confuse if screen reader announces the entire table row information at once.

## Attachments:

https://user-images.githubusercontent.com/100552367/206191003-6e215660-75cc-445e-a69f-5cc5b1be0299.mp4

![NVDA Table](https://user-images.githubusercontent.com/100552367/206191440-e5581bc0-3a7f-48b2-ae5d-d45dae57152c.png)

**Comment by JasonYeMSFT** [comment]
Progress:
- [x] blob
- [ ] file
- [ ] queue
- [ ] table
- [ ] gen2
- [ ] disk
- [ ] app config

**Comment by Jtith** [comment]
issue is still repro unable to navigate inside the table data to get the information.

https://user-images.githubusercontent.com/100552367/211471546-f0d2122a-5543-4bb7-8bf0-67f4c010dfe1.mp4

**Comment by JasonYeMSFT** [comment]
@Jtith I cannot watch the video you posed. Could you upload a gif or post a few screenshots to illustrate how it still repros?

**Comment by Jtith** [comment]
Issue is not repro, hence closing the bug 
![Fixed](https://user-images.githubusercontent.com/100552367/212111682-4a91c999-985c-443e-aa25-fb72b66383fd.png),

**Comment by tripathiakhil429** [comment]
**Github Tags:**#Closed
